### ISSUE TITLE
[Bug] SCAIL Mode: Generated video leads reference pose_images by exactly 4 frames (pose 0-3 are eaten)

### ISSUE THREAD

**Issue by Never2016** [kijai/ComfyUI-WanVideoWrapper#1949]
### Describe the bug
In **SCAIL mode** (`WanVideoAddSCAILPoseEmbeds` + `WanVideoSampler` with SCAIL enabled), the generated video motion **leads the input pose_images by exactly 4 frames**.

- `pose_images[0~3]` are never used in any output frame  
- Output frame 0 always corresponds to `pose_images[4]` (or very close)  
- This happens **regardless of Context windows being on or off**

Animate mode (`WanVideoAnimateEmbeds`) is **perfectly aligned** (frame 0 → pose 0).

<img width="1706" height="665" alt="Image" src="https://github.com/user-attachments/assets/76c736ee-8c3c-4f20-b2ff-3e2b2fe766f3" />

### Steps to reproduce
1. Use the wanvideo_2_1_14B_SCAIL_pose_control_example_01.json workflow
2. Feed a pose sequence (e.g. 81 frames)
3. Compare frame-by-frame: generated video vs reference pose_images
4. Observe: first 4 output frames use pose[4+] information

### Expected behavior
Without any special offset, result frames should be **1:1 aligned** with pose_images (as in Animate mode).

### Actual behavior
SCAIL path exposes an **off-by-one latent frame** (1 latent = 4 pixels) that Animate path compensates for.

### Root cause (code locations)
**Animate path (correctly compensated):**
```python
# wanvideo/modules/model.py
for x_, pose_latents_ in zip(x, pose_latents):
    x_[:, :, 1:].add_(pose_latents_, alpha=strength)   # ← compensates first-frame bias

# wanvideo/modules/model.py
scail_x = [self.patch_embedding_pose(...) for u in scail_pose_latents]
x = [torch.cat([u, v], dim=1) for u, v in zip(x, scail_x)]   # ← direct cat to sequence dim
# No `[:, :, 1:]`, no temporal offset correction
